plugins/woocommerce/src/Blocks/BlockTypes/ProductImage.php

```
@@ -260,7 +260,62 @@ private function render_image( $product, $attributes, $image_id = null ) {
 			$attr['loading'] = $loading_attr;
 		}
 
-		return $provided_image_id_is_valid ? wp_get_attachment_image( $image_id, $image_size, false, $attr ) : $product->get_image( $image_size, $attr );
+		$adjust_srcset = function ( $sources, $size_array, $image_src, $image_meta ) use ( $aspect_ratio ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$size_array'. (undefined)

(UnusedFormalParameter)